src/Backend/Core/Engine/Base/AjaxAction.php

```diff
@@ -41,9 +38,9 @@ public function execute()
      *
      * @return Response
      */
-    public function getContent()
+    public function getContent(): Response
     {
-        $statusCode = (isset($this->content['code']) ? $this->content['code'] : 200);
+        $statusCode = $this->content['code'] ?? self::OK;
```

Comment:
I think it greatly improves readability, and you won't get used to it until you start using it :)